src/pages/blog/starlingx-deploying-your-app-1.md

```diff
@@ -0,0 +1,309 @@
+---
+templateKey: blog-post
+title: Deploying your application to StarlingX
+author: Bruno Muniz
+date: 2023-10-13T12:00:00.042Z
+category:
+  - label: Features & Updates
+---
+
+Deploying your own application to StarlingX is as easy as any other Kubernetes
+deployment out there. In this blog post we'll show you the process with a
+simple demonstration application.<!- more ->
+
+It's important to understand that an application can be deployed in many ways
+to the [Kubernetes cluster(s) that StarlingX manages](https://docs.starlingx.io/operations/k8s_cluster.html)
+:
+
+- [raw Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/);
+- [Helm](https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package);
+- [Flux](https://fluxcd.io/); and finally
+- StarlingX Application, which benefits from tight integration with the
+  [StarlingX system](https://opendev.org/starlingx/config).
+
+In this particular demonstration we will focus on [Helm](https://helm.sh/),
+which is the most popular package manager for Kubernetes. Future blog posts
+will address other deployment types and their advantages.
+
+We will use a virtual All-In-One Simplex (AIO-SX) setup of StarlingX. If you
```

Comment:
Same comment as above with regards to the usage of "we". It is currently not consistent in the blog post whether the reader is supposed to follow along or not. I think it would be beneficial to stick to one approach throughout the article.

If you choose to rather just explain the steps here, you can still add a note towards the end to encourage people to try this out themselves.

Reply:
Makes sense... I'll give more time on this one and review it next week after addressing the other comments.